os/arch/arm/src/armv7-a/arm_cpupause.c

```diff
@@ -97,43 +98,35 @@ static volatile spinlock_t g_cpu_paused[CONFIG_SMP_NCPUS];
 
 bool up_cpu_pausereq(int cpu)
 {
-  return spin_islocked(&g_cpu_paused[cpu]);
+  return spin_is_locked(&g_cpu_paused[cpu]);
 }
 
 /****************************************************************************
- * Name: up_cpu_paused
+ * Name: up_cpu_paused_save
  *
  * Description:
  *   Handle a pause request from another CPU.  Normally, this logic is
```

Comment:
The description here seems duplicated with up_cpu_paused, could you please check?

Reply:
Done. Thanks.